src/electionguard/api/encrypt_ballot.c

```diff
@@ -36,9 +36,6 @@ bool API_EncryptBallot(uint8_t *selections_byte_array,
         selections[i] = selections_byte_array[i] == 1;
     }
 
-    if (!Validate_selections(selections, config.num_selections))
-        ok = false;
-
```

Comment:
Removed extra validate call from here since we'll be calling it anyway in `Voting_Encrypter_encrypt_ballot` 10 lines below here.